### PR TITLE
Fetch our own stars!

### PR DESCRIPTION
**Pull request by TomasHubelbauer**
I'm lumping in some unrelated code formatting, I think `en.json` was updated by the build somehow to remove unused keys? The rest is probably VS Code. Happy to revert formatting to be just the stars.

### CODE REVIEW

**Comment by TomasHubelbauer** [comment]
Oh BTW no styling, maybe later?

**Comment by TomasHubelbauer** [comment]
Closes #24

**Comment by TomasHubelbauer** [comment]
Looks like I didn't break anything obvious hehe

File: website/core/GitHubStarButton.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+}()
+`
+
+class GitHubStarButton extends React.Component {
+  render() {
+    // "Server-side" rendered, so `componentDidMount` wouldn't be called
+    return (
+      <div>
+        <a href="https://github.com/isomorphic-git/isomorphic-git">GitHub <span id="gitHubStars" /></a>
+        <script dangerouslySetInnerHTML={{ __html: script }} />
```
Comment (wmhilton) [inline website/core/GitHubStarButton.js]: Hahaha this is _so_ not the "right" way to populate a React component, but I love it.

**Pull request merged**